NEWS.md

```diff
@@ -0,0 +1,68 @@
+## 1.1.0 - Unreleased
```

Comment:
please update permissions dependency in module descriptor template to 5.6
https://github.com/folio-org/mod-permissions/blob/master/descriptors/ModuleDescriptor-template.json#L7
and also include this story into release notes:
https://issues.folio.org/browse/MODCON-108

Reply:
Added MODCON-108 to news